### ISSUE TITLE
[Hooks] hooks-reference page useReducer section has error in code

### ISSUE THREAD

**Issue by adeelibr** [reactjs/reactjs.org#1643]
Currently in the https://reactjs.org/docs/hooks-reference.html#usereducer section is taking wrong param in `Counter` function.

```diff
- function Counter({initialCount}) {
+ function Counter({initialState}) {
```

**Comment by adeelibr** [comment]
The PR #1642 tackles this issue.

**Comment by saranshkataria** [comment]
closed by #1642
